src/sass/login.scss

```diff
@@ -1,5 +1,202 @@
 @import "shared-variables";
 
+span.sidelines {
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
+  text-align: center;
+  width: 100%;
+  display: inline-block;
+
+  font-size: 1.15em;
+  font-weight: bold;
+
+  &:before, &:after {
+    position: absolute;
+    top: 51%;
+    overflow: hidden;
+    width: 50%;
+    height: 2px;
+    content: "\a0";
+    background-color: $color-gray-light-alt;
+  }
+
+  &:before {
+    margin-left: -52%;
+    text-align: right;
+  }
+
+  &:after {
+    margin-left: 2%;
+  }
+}
+
+.verify, .login {
+  button {
```

Comment:
I've been pushing people on Rainbows to avoid writing styles like this, with generic elements and nesting that's not limited by direct descendent selectors. I think it leads to styles that are hard to understand as they grow and are modified (the claim status styles are good examples of this). And they make styles harder to overwrite, since the nested styles have two or more selectors, which increases specificity. That matters less here, since these pages are small and pretty focused, but it does make a difference for pages with a broader set of features (like the user profile) or in common styles.

The last sass/less codebase I worked on ended up with some selectors that were 7+ classes deep trying to overwrite nested styles that had built up for several years.

Reply:
What's the style you're advocating for? I do think we should be using a lot more of the `>` selector.

Reply:
I have used BEM in the past, but I don't think we need to go that far. I think we should shoot for using single class selectors as much as possible, > selectors when necessary, and almost never style generic elements (aside from site wide widget styles). Some nesting is unavoidable, but I think we should try to minimize it.